diff --git a/code/modules/mob/living/simple_animal/hostile/commanded/guard_dog.dm b/code/modules/mob/living/simple_animal/hostile/commanded/guard_dog.dm
index 27bbf6416b..efac1eb5ea 100644
--- a/code/modules/mob/living/simple_animal/hostile/commanded/guard_dog.dm
+++ b/code/modules/mob/living/simple_animal/hostile/commanded/guard_dog.dm
@@ -148,6 +148,7 @@
 	known_commands = list("stay", "stop", "attack", "follow")
 	attacktext = "licked"
 	attack_sound = 'sound/misc/dog_bark.ogg'
+	faction = "syndicate"
 
 /mob/living/simple_animal/hostile/commanded/dog/iangang/listen(var/mob/speaker, var/text)
     for(var/command in known_commands)
